uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/device/settings/ISettings.java

```diff
@@ -87,4 +88,14 @@ void addWorkAccount(final ITestBroker expectedBroker,
      * UI appears on the device. It will activate the admin for whichever admin requested the activation.
      */
     void activateAdmin();
+
+    /**
+     * Adds screen lock to the device.
+     */
+    void addPinSetup() throws UiObjectNotFoundException;
```

Comment:
Can we rename this to something like `setPinOnDevice`?

Reply:
Also can we accept the PIN as a parameter to this method? Inside the method then we can just set whatever PIN was supplied as the argument

Reply:
I will rename it to setPinOnDevice.

Reply:
So basically when you call this method in the rule, then you will pass in the pin value